Sources/ManageServer.php

```diff
@@ -482,6 +482,12 @@ function ModifyCookieSettings($return_config = false)
 		array('globalCookiesDomain', $txt['globalCookiesDomain'], 'db', 'text', false, 'globalCookiesDomain'),
 		array('secureCookies', $txt['secureCookies'], 'db', 'check', false, 'secureCookies', 'disabled' => !httpsOn()),
 		array('httponlyCookies', $txt['httponlyCookies'], 'db', 'check', false, 'httponlyCookies'),
+		array('samesiteCookies', $txt['samesiteCookies'], 'db', 'select', array(
+				'none' 		=> $txt['samesiteNone'],
+				'lax' 		=> $txt['samesiteLax'],
+				'strict' 	=> $txt['samesiteStrict']
+			),
+			'samesiteCookies'),
```

Comment:
The documentation states that SameSite=None must be "Secure".  I would add a save logic here that if secure is checked and samesite is set to none, to set it back to lax.

Reply:
im aware of this, but i see many edge cases which get to complex for me to handle.
Feel free to provide implementation or do it later on.